test/unit/io.js

```diff
@@ -861,6 +861,54 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression does not throw error', function () {
```

Comment:
Could you suggest a strategy for the positive tests? I don't want to simply test that the file got smaller, since occasionally that doesn't work (16 bit tiff files with zip lzw have issues sometimes). Is there a way to read from the metadata that a tiff is compressed?